datastacks/datastacks_cli.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import shutil
+import yaml
+
+import click
+from click_loglevel import LogLevel
+
+from datastacks.logger import setup_logger
+from datastacks.utils import render_template_component
+
+
+@click.group()
+@click.option("--log-level", "-l", type=LogLevel(), default=logging.INFO)
+def cli(log_level):
+    setup_logger("datastacks", log_level)
+
+
+@click.command()
+@click.option("--config-path", "-c", type=str, help="Absolute path to config file on local machine")
```

Comment:
Presume the README / documentation will have the required structure of the config file?

Reply:
This is included in the readme which has been added. Have also left a sample config file